### DISCUSSION TITLE
Problem is configration.

### DISCUSSION THREAD

**Discussion by VimYoung** [tauri-apps/tauri#9354] [Q&A] [answered]
I have been trying to install the AppImage of chatgpt from [here](https://github.com/lencx/ChatGPT).
Since the application image was not showing in the applications bar in ubuntu, I refered to [this](https://davidmatthew.ie/how-to-launch-an-appimage-ubuntu/) article to find a work around.
On running the following command at last, it was doing nothing:
`sudo desktop-file-install chat-gpt.desktop`
(I created the `chat-gpt.desktop` myself accourding to the article)
When nothing happened I executed the exec variable in the file directly on the command prompt to check for the problem, the command was as follows
`/opt/chat-gpt/AppRun %F` 
This is the same AppRun file which I received from the AppImage which i downloaded from the mentioned github page.
It is giving me the following error now, And Chat-gpt(browser): suggested me to ask here. Any help or redirections would be appreciable.
```
thread 'main' panicked at '`APPDIR` or `APPIMAGE` environment variable found but this application was not detected as an AppImage; this might be a security issue.', /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tauri-utils-1.3.0/src/lib.rs:205:11
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

and then this after `RUST_BACKTRACE=1` is set
```
thread 'main' panicked at '`APPDIR` or `APPIMAGE` environment variable found but this application was not detected as an AppImage; this might be a security issue.', /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tauri-utils-1.3.0/src/lib.rs:205:11
stack backtrace:
   0:     0x64f356f5f494 - <unknown>
   1:     0x64f356df62ef - <unknown>
   2:     0x64f356f32220 - <unknown>
   3:     0x64f356f6100f - <unknown>
   4:     0x64f356f60895 - <unknown>
   5:     0x64f356f617a4 - <unknown>
   6:     0x64f356f61263 - <unknown>
   7:     0x64f356f611f6 - <unknown>
   8:     0x64f356f611e1 - <unknown>
   9:     0x64f356c46f22 - <unknown>
  10:     0x64f356d12e9b - <unknown>
  11:     0x64f356ce83cf - <unknown>
  12:     0x64f356daebda - <unknown>
  13:     0x64f356d3a707 - <unknown>
  14:     0x64f356dae7f6 - <unknown>
  15:     0x7dd19be29d90 - __libc_start_call_main
                               at ./csu/../sysdeps/nptl/libc_start_call_main.h:58:16
  16:     0x7dd19be29e40 - __libc_start_main_impl
                               at ./csu/../csu/libc-start.c:392:3
  17:     0x64f356c5abd5 - <unknown>
  18:                0x0 - <unknown>
```

**Comment by FabianLars** [comment]
The appimage you linked is too old. We changed the panic to a simple warning, so if the projects updates the tauri version and rebuild the appimage it should not crash with this error anymore.

**Reply by VimYoung** [reply]
Hey @FabianLars, Is there any work around, or seems like I would need to continue using the browser to access gpt 😁😅

**Reply by FabianLars** [reply, marked as answer]
hmm, i can't think of another way other then the aforementioned rebuild of the appimage.
